Documentation/v2/04_to_2_snapshot_migration.md

```diff
@@ -1,4 +1,4 @@
-# Snapshot Migration
```

Comment:
probably add a header is good enough?

something like **This is the documentation for etcd2 releases. Read [etcd3 doc][v3] for etcd3 releases**